example-model/src/main/java/com/appunite/rx/example/model/dao/PostsDao.java

```diff
@@ -78,6 +87,19 @@ public PostsDao(@Nonnull final Scheduler networkScheduler,
                     }
                 });
 
+
+                sendPost.flatMap(new Func1<AddPost, Observable<ResponseOrError<Response>>>() {
+                    @Override
+                    public Observable<ResponseOrError<Response>> call(AddPost post) {
+                        return guestbookService.createPost(post).subscribeOn(networkScheduler)
+                                .compose(ResponseOrError.<Response>toResponseOrErrorObservable());
+                    }
+                })
+                        .observeOn(uiScheduler)
+                        .compose(MoreOperators.<ResponseOrError<Response>>cacheWithTimeout(uiScheduler))
```

Comment:
you don't need to use cacheWithTimeout - you don't have to cache enything if you subscribe only once